test/Pipfile

```diff
@@ -6,59 +6,10 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-apipkg = "==1.5"
-atomicwrites = "==1.4.1"
-attrs = "==19.3.0"
-bcrypt = "==3.1.7"
-cached-property = "==1.5.1"
-certifi = "==2019.11.28"
-cffi = "==1.13.2"
-chardet = "==3.0.4"
-configparser = "==4.0.2"
-contextlib2 = "==0.6.0.post1"
-coverage = "==5.0.1"
-cryptography = "==3.3.2"
-docker = "==4.1.0"
-dockerpty = "==0.4.1"
-docopt = "==0.6.2"
-enum34 = "==1.1.6"
-execnet = "==1.7.1"
-filelock = "==3.0.12"
-funcsigs = "==1.0.2"
-idna = "==2.8"
-importlib-metadata = "==1.3.0"
-ipaddress = "==1.0.23"
-jsonschema = "==3.2.0"
-more-itertools = "==5.0.0"
-pathlib2 = "==2.3.5"
-pluggy = "==0.13.1"
-py = "==1.10.0"
-pycparser = "==2.19"
-pyparsing = "==2.4.6"
-pyrsistent = "==0.15.6"
-pytest = "==4.6.8"
-pytest-cov = "==2.8.1"
-pytest-forked = "==1.1.3"
-pytest-xdist = "==1.31.0"
-requests = "==2.28.1"
-scandir = "==1.10.0"
-six = "==1.13.0"
-subprocess32 = "==3.5.4"
-testinfra = "==3.3.0"
-texttable = "==1.6.2"
-toml = "==0.10.0"
-tox = "==3.14.3"
-urllib3 = "==1.26.5"
-virtualenv = "==16.7.9"
-wcwidth = "==0.1.7"
-zipp = "==0.6.0"
-"backports.shutil_get_terminal_size" = "==1.0.0"
-"backports.ssl_match_hostname" = "==3.7.0.1"
-Jinja2 = "==2.11.3"
-MarkupSafe = "==1.1.1"
-PyYAML = "==5.4"
-websocket_client = "==0.57.0"
-python-dotenv = "==0.17.1"
+pytest = "==7.1.3"
+pytest-xdist = "==2.5.0"
+pytest-testinfra = "==6.8.0"
```

Comment:
`testinfra` was migrated over to pytest in `6.0.0`: https://testinfra.readthedocs.io/en/latest/changelog.html#id9